### PR TITLE
Refactor code -  Add boilerplate code for supported languages

### PR DESCRIPTION
**Pull request by pavanskipo**
Hello! 👋

This PR has the following changes

- Adds boilerplate code for supported languages
image

- Refactor Code and moved `repl_modules.sh` into `modules` folder where we have the boilerplates.

The main idea is that in future we can add a new command say `repl config boilerplate <language>`, to allow user to define their own starter code whenever they create a new repl.

### CODE REVIEW

File: modules/boilerplates/repl.go [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package main
+import ("fmt")
+
+func main() {
+  fmt.Println("Hello World! whats up")
```
Comment (DamianRivas) [inline modules/boilerplates/repl.go]: Could you remove the ` whats up` at the end here?

**Review by DamianRivas — CHANGES_REQUESTED** [review]
Excellent work here! I'm requesting changes for consistency, but overall this is great thank you.

For the `repl.rb` boilerplate, we may as well add `puts 'Hello World!'` to that module.

With these boilerplate modules, we can remove the code like the following from `test.sh`. These echo statements populated the hello world code into the test files which isn't necessary anymore now that we have the boilerplates 🎉 
```bash
  echo "puts 'Hello world!'" > repl_tmp/repl.rb
  assertContains "$(cat repl_tmp/repl.rb)" "Hello world!"
```

File: modules/boilerplates/repl.go (hunk 1 above)
Comment (pavanskipo) [reply modules/boilerplates/repl.go]: > 

@DamianRivas Makes sense, updated!

File: test/modules_test.sh [hunk 2]
```diff
@@ -20,31 +20,21 @@ setUp() {
   repl_update_config "TMP=" "repl_tmp"
 }
 
-assert_repl_run() {
-  assertContains "$(bin/repl run)" "Hello world!"
-}
-
```
Comment (DamianRivas) [inline test/modules_test.sh]: We actually want to keep this. This is what tests the `repl run` command

File: test/modules_test.sh [hunk 3]
```diff
@@ -73,29 +59,21 @@ test_rb_module() {
 
   echo "puts 'Hello world!'" > repl_tmp/repl.rb
```
Comment (DamianRivas) [inline test/modules_test.sh]: These `echo` statements that put the `Hello world!` programs into the files are the redundancies we can remove now thanks to the boilerplates

Comment (DamianRivas) [reply test/modules_test.sh]: Additionally, now that all modules have the `Hello world!` boilerplate, we can add the `assertContains` check to all tests that makes sure the files generated by repl contain `Hello world!`

**Review by DamianRivas — CHANGES_REQUESTED** [review]

File: test/modules_test.sh (hunk 3 above)
Comment (pavanskipo) [reply test/modules_test.sh]: hello! Updated code and added the additional check.

**Review by DamianRivas — APPROVED** [review]
Thank you 🎉

**Pull request merged**